src/unix/linux_like/linux/arch/generic/b32.rs

```diff
@@ -0,0 +1,39 @@
+// use core::mem::size_of;
+
+// sizeof(time_t) == sizeof(__kernel_long_t)
```

Comment:
If we turn off this, then I don't think we should leave them?

Reply:
Actually I think it should be turned on, but some of the kernels in CI doesn't support that. I left them here because sometime in the future we will need to turn it on.

Reply:
Still, I'd like to remove it until we can actually import them to simplify code, our structure is complex currently so I don't want to make it any more complicated unless we have to.

Reply:
Ok.